language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/FieldAccessContext.java

```diff
@@ -52,4 +53,16 @@ public FieldAccessContext(Class<T> attachmentPoint) {
 
         return this.getCompletionItemList(symbolList, ctx);
     }
+
+    @Override
+    public void sort(BallerinaCompletionContext context, T node, List<LSCompletionItem> completionItems) {
+        // First we do the default sorting
+        super.sort(context, node, completionItems);
```

Comment:
Here we traverse the completion item list twice. In this particular context we can traverse the list once and assign the priorities. or, in a more configurable way, we can add a sort API to the sorting utils to accept a priority offset and assign the priority a completion item by offsetting the default priority.